fix: [MTM-58369] add missing release notes for UI (#1870)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-10-18-291-0-clear-functionality-under-usage-statistics-is-not-working-as-expected.md

---

> date: 24.03.2024

Reviewer: Please make sure it's the correct date format.

> title: Issue with clear button in usage statistics

Reviewer: ```suggestion
title: Improved clearing of filters on Usage statistics page
```
